pkgs/applications/blockchains/zcash/default.nix

```diff
@@ -20,15 +20,16 @@ rustPlatform.buildRustPackage.override { inherit stdenv; } rec {
       --replace "linker = \"aarch64-linux-gnu-gcc\"" ""
   '';
 
-  cargoHash = "sha256-6uhtOaBsgMw59Dy6yivZYUEWDsYfpInA7VmJrqxDS/4=";
+  cargoHash = "sha256-/KBgUrvxfGlbY+9XVYASwYZUxbhW5q43NTBsvUZSkuo=";
 
-  nativeBuildInputs = [ autoreconfHook cargo cxx-rs hexdump makeWrapper pkg-config ];
+  nativeBuildInputs = [ autoreconfHook cargo cxx-rs git hexdump makeWrapper pkg-config ];
 
   buildInputs = [
     boost180
     db62
     libevent
     libsodium
+    tl-expected
```

Comment:
@teor2345 I just noticed... `tl-expected` is _only_ a header file. As such, it is a build time dependency, not a runtime dependency. Would it be more appropriate to list it as a `nativeBuildInput` rather than a `buildInput`?

Reply:
Fixed in 83f85a7

Reply:
> Remove header-only dependency tl-expected from nativeBuildInputs

Why was this action taken?

Reply:
It is easier if we stuff all libraries into buildInputs. Also in the future the package might change and be a single header file plus a script which then could be for the wrong architecture.